### PR TITLE
Feature/update footer

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

File: src/locale/properties/layout/layout.en.properties [hunk 1]
```diff
@@ -6,10 +6,11 @@ common.old_browser_3=supported browser
 footer.copyright_cc0_1=The text of this website is published under a
 footer.copyright_cc0_3=Images and marks are subject to copyright and trademark protection
 common.period=.
-footer.aboutOrcid=About ORCID
 footer.privacypolicy=Privacy Policy
 footer.termsofuse=Terms of Use
 footer.contactus=Contact us
+footer.accessibility=Accessibility Statement
```
Comment (DanielPalafox) [inline src/locale/properties/layout/layout.en.properties:12]: The new properties are missing in the testing languages

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

File: src/locale/properties/layout/layout.en.properties (hunk 1 above)
Comment (leomendoza123) [reply src/locale/properties/layout/layout.en.properties:12]: CI is supposed to generate those let's wait to see if it does

**Comment by leomendoza123** [comment]
[Registry changes needed for info site migration (5)](https://trello.com/c/OIlbX1qu/6829-registry-changes-needed-for-info-site-migration-5)

**Review by DanielPalafox — APPROVED** [review]

**Comment by leomendoza123** [comment]
[The footer ULRs of the old and new app does not match](https://trello.com/c/39u6SNBY/7065-the-footer-ulrs-of-the-old-and-new-app-does-not-match)

**Pull request merged**